daemon/manager/manager.go

```diff
@@ -101,17 +101,23 @@ func Execute(configs *options.BootstrapConfigs) error {
 	log.Info("command Start cni successful")
 	defer cniInstaller.Stop()
 
-	setupCloseHandler()
+	*configs.Status = setupCloseHandler()
+	log.Printf("bpfLoader.Restart %v", *configs.Status)
 	return nil
 }
 
-func setupCloseHandler() {
+func setupCloseHandler() int {
 	ch := make(chan os.Signal, 1)
 	signal.Notify(ch, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT, syscall.SIGHUP, syscall.SIGABRT, syscall.SIGTSTP)
 
 	<-ch
 
 	log.Warn("exiting...")
+	
+	if bpf.GetDaemonset() {
```

Comment:
Why if we get Kmesh daemonset object, we could make sure it's restart?

When Kmesh is create for the first time, it always creates the Kmesh DaemonSet object first and then creates the Kmesh pods.

Reply:
yes, always creates the Kmesh DaemonSet object first and then creates the Kmesh pods.
and by end, there is some confusion，
although the current I can judge if restart by daemonset in bpfLoader.Stop, there is still a lack of theoretical basis